kiota_abstractions/request_information.py

```diff
@@ -155,18 +159,26 @@ def set_content_from_parsable(
             content_type (Optional[str]): the content type.
             values (Union[T, List[T]]): the models.
         """
-        writer = self._get_serialization_writer(request_adapter, content_type, values)
-
-        if isinstance(values, list):
-            writer.write_collection_of_object_values(None, values)
-        else:
-            writer.write_object_value(None, values)
+        span = self.start_tracing_span("set_content_from_parsable")
+        try:
+            writer = self._get_serialization_writer(
+                request_adapter, content_type, values, span
+            )
+
+            if isinstance(values, list):
+                writer.write_collection_of_object_values(None, values)
+            else:
+                writer.write_object_value(None, values)
```

Comment:
And here `span.set_attribute(REQUEST_TYPE_KEY, "object")`

Reply:
What's the actual string for REQUEST_TYPE_KEY?

Reply:
"com.microsoft.kiota.request.type" seems to be the most common value across the sdks